docs/src/content/tutorial/workflow.md

```diff
@@ -0,0 +1,101 @@
+
+## Linked Data
+
+[Tim Berners Lee defines](https://www.w3.org/DesignIssues/LinkedData.html) Linked data as:
+
+> The Semantic Web isn't just about putting data on the web. It is about making links, so that a person or machine can explore the web of data. With linked data, when you have some of it, you can find other, related, data.
+
+There are three rules that the linked data should follow:
+
+1. Use URI for names of things
+2. Use HTTP URIs so that people can look up those names.
+3. When someone looks up a URI, provide useful information, using the standards.
+4. Include links to other URIs. so that they can discover more things.
+
+With Linked data it becomes easy to integrate different datasets together to achieve different goals. One of the common examples of Linked data dataset is [DBPedia](https://wiki.dbpedia.org/)(the dataset of wikipedia in Linked data format).
+
+## JSON-LD
+
+[JSON-LD](https://json-ld.org/) is a JSON based format to serialise linked data. It annotates the elements of the response and structures the data which helps to disambiguate different data and helps machines better understand the data.
+For example, any human can comprehend that the following response contains data about the name Hydra Ecosystem and contains the url of home page. But machines cannot make any sense from the following data.
+
+```json
+{
+  "name": "Hydra Ecosystem",
+  "homepage": "https://hydraecosystem.org/"
+}
+```
+
+To resolve this, one can use unambiguous identifiers for the name and URL and specify that if the value is an IRI or a string literal. JSON-LD can be used to achieve this by using a popular schema.org vocabulary:
+
+```json
+{
+    "http://schema.org/name": "Hydra Ecosystem",
+    "http://schema.org/url": {
+        "@id": "https://hydraecosystem.org/"
+        ↑ The '@id' keyword means 'This value is an identifier that is an IRI
+    }
+}
+```
+
+## Web APIs
+
+A Web API is an application programming interface for either a web server or a web browser. One can get the data without bothering about the implementation details. For example to get the weather information of a place, one need not to bother about how the weather is measured. A request to the URL of the API returns with the weather information.
+
+<div id="main">
+After going through this document, you should be able to understand the general workflow followed to create and use APIs using the tools of the Hydra Ecosystem. Though the workflow might be different for few use cases, this document tries to cover the general workflow that can satisfy most of the use cases.
+</div>
+
+To design web APIs using the tools of this ecosystem, follow three simple steps.
+
+1. Define the API in JSON-LD format as API Documentation
+2. Pass this API Documentation to the `hydrus` server
+3. Use Smart clients(`hydra-python-agent`) to consume APIs served by the `hydrus`.
+
```

Comment:
Hmm, when we are keeping the heading of this article is `General Workflow`, I think the reader should directly get access to the workflow without first getting interrupted by having to go through the related terminologies.

I understand that these terminologies are very important to understand the workflow so how about we probably move all these to another page(which has only terminologies/glossary) which we can hyperlink here?

Reply:
The very first paragraph contains the note with a link to the main content, if they are familiar with these, they can jump directly there. That being said, I was not convinced of the idea to direct people to go through a glossary first. I think that will be a little overwhelming. Better to wait to see what other reviewers think about this!